### DISCUSSION TITLE
Lesson-6: Error:  (code: -32003, message: Insufficient funds for gas * price + value, data: None)

### DISCUSSION THREAD

**Discussion by Perso77** [Cyfrin/foundry-full-course-cu#44] [Q&A] [answered]
i try to broadcast with RPC ganache but not working! under the pretext that: Insufficient funds for gas * price + value, data: None ?
![simplestorage](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/f88bfb6d-3528-4294-bd07-ffc82767d142)
![ganache A](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/3546f71f-495b-4cac-a34d-eb5afe3653da)
!!

**Comment by ghost** [comment, marked as answer]
Hello @Perso77, the RPC ULR you're passing to the command line is from Anvil. 

Ganache has a different RPC URL `http://172.20.112.1:7545`, check the top of the UI. Also make sure you're using a private key from Ganache.

**Reply by Perso77** [reply]
Yes, I haven't paid attention, thanks

**Reply by cromewar** [reply]
Nice catch @pacelliv
